nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -3348,7 +3337,7 @@ to the cluster. It provides an additional layer of security. This value is blank
 long time before starting processing if we reach at least this number of nodes in the cluster.
 |`nifi.cluster.load.balance.port`|Specifies the port to listen on for incoming connections for load balancing data across the cluster. The default value is `6342`.
 |`nifi.cluster.load.balance.host`|Specifies the hostname to listen on for incoming connections for load balancing data across the cluster. If not specified, will default to the value used by the `nifi.cluster.node.address` property.
-|`nifi.cluster.load.balance.connections.per.node`|The maximum number of connections to create between this node and each other node in the cluster. For example, if there are 5 nodes in the cluster and this value is set to 4, there will be up to 20 socket connections established for load-balancing purposes (5 x 4 = 20). The default value is `4`.
```

Comment:
Still seems to be 4: https://github.com/apache/nifi/blob/master/nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/pom.xml#L200

Reply:
Thanks for reviewing @pvillard31 ! The default values need to be changed via https://issues.apache.org/jira/browse/NIFI-7308.

Reply:
Hey @andrewmlim - do you want to make the changes related to NIFI-7308 in this PR? or do we want another PR? I think it's better to merge the doc changes once the code has been updated.

Reply:
I agree the code change should be done before these doc changes are put in. I was thinking a separate PR would be created for NIFI-7308.

Reply:
Have confirmed that the code changes have been made for everything except changing the default value for nifi.provenance.repository.index.shard.size to be 250, but confirmed with @markap14 that the current default value of 500 is fine. So need to make some changes to this PR in addition to needing to merge to main branch not master.

Thinking I should just close this PR and submit a new one.